master/buildbot/test/util/db.py

```diff
@@ -135,7 +135,11 @@ def __thd_clean_database(self, conn):
 
             # Drop all reflected tables and indices. May fail, e.g. if
             # SQLAlchemy wouldn't be able to break circular references.
-            meta.drop_all()
+            # Sqlalchemy fk support with sqlite is not yet perfect, so we must deactivate fk during that
+            # operation, even though we made our possible to use use_alter
+            with withoutSqliteForeignKeys(conn.engine):
+                meta.drop_all()
```

Comment:
as sqlite does not support alter table, migrate does  renames the table, and then the fk contraints are renamed too, and then you cannot destroy the renamed table.
The code is here. It supposed to do the correct thing, but this does not work for me, and gives a lot of "migration_tmp does not exist" errors
https://github.com/openstack/sqlalchemy-migrate/blob/master/migrate/changeset/databases/sqlite.py